### DISCUSSION TITLE
Indoors timelapse for 30 days w/ 6 x HERO10 and upload?

### DISCUSSION THREAD

**Discussion by datdll** [gopro/labs#910] [Q&A]
I have a project coming up to create a timelapse of **1 minute** for a team that will be building a puzzle for **8 hours a day for 1 month** straight. I am going to mount **6 x HERO10** on the wall put them and plug them on A/C for constant power. I intend on installing the Labs firmware and shoot for _10 hours a day, 24fps, every 10 minutes which is about 1440 images_ per my calculations for each GoPro. I would appreciate upload on Google Drive after every capture aswell, WiFi is available on location. I will remove the batteries as I've seen on other posts here and keep only the A/C and I will GPS sync all 6 of them. Can I create a QR with all these commands once? Thoughts on this? Will it work as intended? Any tips tricks or anything at all you can help me with please let me know as I have never done this before. Thank you so much.

**Bullet points for ease,**

- Which GoPro: HERO 10 x 6
- Final Clip Length: 1 minute
- Shooting Time: 30 days mounted & GoPro's active for 8-10 hours per day
- A/C & WiFi available: Yes
- Upload on Google Drive: Yes if possible

**Comment by dnewman-gpsw** [comment]
Don't remove the battery as that reduces the reliability, battery is useful in keeping the internal clock accurate.  HERO11 or 12 is likely more reliable, although there is a HERO10 update coming soon, which may improve that. Auto upload only works to the GoPro cloud, it will also reduce reliability if the Wi-Fi connection if slow or unreliable. You my want to stagger the upload times. GPS sync shouldn't be required. As you as planning on A/C powering, you just need to start the timelapse like ```!09:00NmT!S!17:00E!2U!R```

Do lots of testing.

**Reply by datdll** [reply]
Thank you for your reply. So keep the battery on? Can I buy the cloud then and use the same one for all 6 HERO10? I think the WiFi is good right there but I want to make sure I can keep track of whats going on and  I prefer to have everything uploaded on cloud daily. Is it possible to check via the App the status of each HERO10 on site if they are working? Will all be installed on high ceiling.

Also I have like  limited day to test. Fingers crossed

**Reply by dnewman-gpsw** [reply]
Multiple cameras do sync with one GoPro account.   I don't know if there is a limit,  I expect not.  This not very much data.

**Reply by datdll** [reply]
Any other tips or tricks you can share with a noob?
